api/graphql/schema.graphql

```diff
@@ -155,6 +155,7 @@ type User {
 type CommunityInterest {
   id: UUID!
   community: Community! @belongsTo
+  user: User! @belongsTo @canRoot(ability: "viewUser")
```

Comment:
The `authorizedToView` scope makes this policy/authorization call not needed at the moment, but I thought it was best to include preemptively to avoid accidental access to user models